### PR TITLE
Increase test coverage: Array.prototype.toString

### PR DESCRIPTION
**Pull request by matedabis**
The coverage is measured by the following script, using `--jerry-test-suite --test262 --unittests --jerry-tests`:
https://github.com/matedabis/jerryscript/blob/gcov_coverage_tester/tests/gcov-tests/gcovtester.py

Branch coverage:
Before: 5/6
After:  6/6

JerryScript-DCO-1.0-Signed-off-by: Mate Dabis mdabis@inf.u-szeged.hu

### CODE REVIEW

File: tests/jerry/array-prototype-tostring.js [hunk 1]
```diff
@@ -61,3 +61,10 @@ try {
   assert (e.message === "foo");
   assert (e instanceof ReferenceError);
 }
+
+// Checking behavior when undefined is passed to the function
+try {
+  Array.prototype.toString.call(undefined);
```
Comment (rerobika) [inline tests/jerry/array-prototype-tostring.js]: Add an `assert(false)` after this line.

File: tests/jerry/array-prototype-tostring.js [hunk 2]
```diff
@@ -61,3 +61,10 @@ try {
   assert (e.message === "foo");
   assert (e instanceof ReferenceError);
 }
+
+// Checking behavior when undefined is passed to the function
```
Comment (rerobika) [inline tests/jerry/array-prototype-tostring.js]: At this point not an `undefined` passed to the function. The function's `this_argument` is `undefined`.

**Comment by matedabis** [comment]
I updated the patch according to the review.

**Comment by matedabis** [comment]
Updated the patch:
- Added comment for the case containing which part of the ES5.1 it tests.
- Ran the test code in Gecko, V8, SpiderMonkey engines and got the same result.

File: tests/jerry/array-prototype-tostring.js [hunk 3]
```diff
@@ -61,3 +61,12 @@ try {
   assert (e.message === "foo");
   assert (e instanceof ReferenceError);
 }
+
+/* ES 5.1 15.4.4.2 1. */
+// Checking behavior when the function's this_argument is undefined
```
Comment (rerobika) [inline tests/jerry/array-prototype-tostring.js]: If the related part of the standard is present, than use the following format:

```
/* ES v5.1, 15.4.4.2.1. 
   Checking behavior when the function's this_argument is undefined */
```

**Comment by matedabis** [comment]
Changed comment style.

**Comment by matedabis** [comment]
Closing PR because of moving to another branch.

**Pull request closed without merging**